src/views/repository/content/BasesFormText.vue

```diff
@@ -0,0 +1,279 @@
+<template>
+  <UnnnicSkeletonLoading
+    v-if="item.status === 'loading'"
+    tag="div"
+    height="100%"
+    class="repository-base-edit__wrapper__card-content"
+  />
+
+  <div
+    v-else
+    :class="[
+      'repository-base-edit__wrapper__card',
+      'repository-base-edit__wrapper__card-content',
+    ]"
+  >
+    <div class="repository-base-edit__wrapper__card-content__header">
+      {{ $t('content_bases.write_content') }}
+
+      <UnnnicButton
+        :loading="item.status === 'saving'"
+        @click="saveText"
+        size="small"
+        class="repository-base-edit__wrapper__card-content__header__save-button"
+        :disabled="!item.value.trim() || item.value === item.oldValue"
+      >
+        {{ $t('webapp.settings.save') }}
+      </UnnnicButton>
+    </div>
+
+    <textarea
+      v-model="item.value"
+      name=""
+      id="textId"
+      cols="30"
+      rows="10"
+      class="repository-base-edit__textarea"
+      :placeholder="$t('content_bases.write_content_placeholder')"
+    ></textarea>
+
+    <div
+      v-if="!item.value.trim() && false"
+      class="repository-base-edit__wrapper__card-content__info"
+    >
+      <UnnnicIcon
+        icon="help"
+        size="sm"
+        scheme="neutral-cloudy"
+      />
+
+      <span v-html="$t('content_bases.write_content_help')"></span>
+    </div>
+  </div>
+</template>
```

Comment:
Improve the semantics of this component

Reply:
It really needs it, because it comes from an old code.

Reply:
```suggestion
<template>
  <UnnnicSkeletonLoading
    v-if="item.status === 'loading'"
    tag="div"
    height="100%"
    class="repository-base-edit__wrapper__card-content"
  />

  <section
    v-else
    :class="[
      'repository-base-edit__wrapper__card',
      'repository-base-edit__wrapper__card-content',
    ]"
  >
    <header class="repository-base-edit__wrapper__card-content__header">
      {{ $t('content_bases.write_content') }}

      <UnnnicButton
        :loading="item.status === 'saving'"
        @click="saveText"
        size="small"
        class="repository-base-edit__wrapper__card-content__header__save-button"
        :disabled="!item.value.trim() || item.value === item.oldValue"
      >
        {{ $t('webapp.settings.save') }}
      </UnnnicButton>
    </header>

    <textarea
      v-model="item.value"
      name=""
      id="textId"
      cols="30"
      rows="10"
      class="repository-base-edit__textarea"
      :placeholder="$t('content_bases.write_content_placeholder')"
    ></textarea>

    <section
      v-if="!item.value.trim() && false"
      class="repository-base-edit__wrapper__card-content__info"
    >
      <UnnnicIcon
        icon="help"
        size="sm"
        scheme="neutral-cloudy"
      />

      <p v-html="$t('content_bases.write_content_help')"></p>
    </section>
  </section>
</template>
```